pkg/tbtc/node.go

```diff
@@ -25,9 +28,16 @@ func newNode(
 	netProvider net.Provider,
 	persistence persistence.Handle,
 ) *node {
+	// TODO: Pass TSS pre-parameters pool config from the outside.
```

Comment:
Watch out for the changes from #3090. This PR altered how configuration is passed.